### ISSUE TITLE
Feature: Add the community chat support[both frontend + backend]

### ISSUE THREAD

**Issue by arunimaChintu** [Kajol-Kumari/InfluMart#197]
Add the community chat support[both frontend + backend]

**Comment by RohitGupta1237** [comment]
@arunimaChintu  can u work on this? how much time would u take to do this

**Comment by arunimaChintu** [comment]
@RohitGupta1237 yes, I want to work on this issue... Please assign me this issue... 1 day

**Comment by arunimaChintu** [comment]
@RohitGupta1237 can you please assign the gssoc label and levels
